qmxgraph/common_testing.py

```diff
@@ -32,3 +33,31 @@ def get_cell_ids(widget, filter_function):
         return cells.map(function(aCell){{ return aCell.getId(); }});
     }})()'''.format(filter_func=filter_function))
     return cells_ids
+
+
+@contextmanager
+def bridge_blocker(*bridge_signal_list):
```

Comment:
```suggestion
def wait_signals(*signals: pyqtSignal):
```

This is generic and not related to bridges specifically correct?